salt/utils/process.py

```diff
@@ -807,11 +807,15 @@ def _handle_signals(self, signum, sigframe):
         msg += '. Exiting'
         log.debug(msg)
         if HAS_PSUTIL:
-            process = psutil.Process(self.pid)
-            if hasattr(process, 'children'):
-                for child in process.children(recursive=True):
-                    if child.is_running():
-                        child.terminate()
+            if psutil.pid_exists(self.pid):
```

Comment:
I'd avoid checking for this, because it can mean nothing. What happens if gets stopped after the check?

I'd add the try-except block around the original code and that's it.

Reply:
adjusted try/except block